src/helpers/connectWalletHelper.ts

```diff
@@ -0,0 +1,27 @@
+import * as w2wHelper from 'helpers/w2w';
+
+export const retrieveUserPGPKeyFromStorage = (account: string) => {
```

Comment:
```suggestion
export const retrieveUserPGPKeyFromStorage = (account: string | null) => {
```
Since the function also returns null